test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
```

Comment:
Those are version numbers, not numeric values, we use U+002D HYPEN-MINUS (which also indicates subtraction), not U+2013 EN DASH, and this is a [formal support comment](https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-69379197), not English prose.